src/components/messenger/list/utils.tsx

```diff
@@ -11,3 +11,11 @@ export function lastSeenText(user): string {
 
   return 'Last Seen: ' + (user.lastSeenAt ? fromNow(user.lastSeenAt) : 'Never');
 }
+
+export function escapeRegExp(string) {
```

Comment:
`import escapeRegExp from "lodash/escapeRegExp"`

Maybe we should use lodash? I imagine they'd maintain this better than we would.

Reply:
i didn't knew it existed. Thanks!